crates/router/src/utils.rs

```diff
@@ -1324,3 +1325,14 @@ pub async fn trigger_refund_outgoing_webhook(
 ) -> RouterResult<()> {
     todo!()
 }
+
+pub fn get_locale_from_header(headers: Option<&actix_web::http::header::HeaderMap>) -> String {
```

Comment:
Any specific reason for making the parameter optional?

We can keep it as a HeaderMap itself!